src/index.css

```diff
@@ -88,6 +91,10 @@ body {
     color: var(--tertiary);
     text-decoration: underline;
   }
+  .hover-menu-element:hover {
```

Comment:
Prevents `hover` state not disappear on tap on menu (basically doesn't apply hover at all)

Reply:
I tried using `:active`, but it doesn't work cool, I think mobiles doesn't have a hover at all, so we don't need to apply it for mobiles